doc/index.rst

```diff
@@ -0,0 +1,62 @@
+.. raw:: html
+
+   <style type="text/css">
+   .thumbnail {{
+       position: relative;
+       float: left;
+       margin: 10px;
+       width: 180px;
+       height: 200px;
+   }}
+
+   .thumbnail img {{
+       position: absolute;
+       display: inline;
+       left: 0;
+       width: 170px;
+       height: 170px;
+   }}
+
+   </style>
+
+pyRiemann-qiskit: Qiskit wrapper for pyRiemann
+=============================================================
+
+pyRiemann-qiskit is a Qiskit wrapper around pyRiemann. It allows to use quantum classification with riemannian geometry. 
```

Comment:
Can you complete this very short introduction? especially for persons like me who know absolutely nothing about quantum computing.

For example:

- What are the interests of using pyRiemann-qiskit, instead of pyRiemann? Is it a faster computational time? Better classification accuracies? Both? Other properties?

- What are hardware requirements? a quantum computer? etc

This introduction should also be copied at the beginning of the readme file of the repo. And the readme file should make reference to 3 or 4 major publications.